src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,120 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  width: $product-card-width;
+  background-color: white;
+  padding: 16px;
+  text-align: center;
+
+  transition: transform 300ms;
+
+  &__photo {
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
+  }
+
+  .description {
+    margin-top: 36px;
+    text-align: start;
+
+    &__title {
+      font-weight: $normal-font;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &__product-code {
+      margin-top: 4px;
+      font-size: 10px;
+      line-height: 14px;
+      font-weight: $light-font;
+      color: $lighter-text-color;
+    }
+  }
+
+  .stars {
+    display: flex;
+
+    &__star {
+      width: $star-size;
+      height: $star-size;
+      background-repeat: no-repeat;
+      background-image: url(../images/star.svg);
+      background-position: center;
+
+      margin-right: $stars-gap;
+
+      &:nth-child(-n + 4) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
+  }
+
+  .rating {
```

Comment:
make it an element of card block. same for `price-string` (bad name BTW - use `card__price-wrapper`, etc)

Reply:
Note: you can have nested into each other elements

For example, card__rating can be an element of the card block, and card__reviews can be an element of the card block. And it's allowed you to put card__reviews inside of card__rating in HTML structure